src/common/view-definitions/version-4/legend-definition.ts

```diff
@@ -22,17 +24,49 @@ export interface LegendDefinition {
   hasNull: boolean;
 }
 
+const dimensionKindToPlywoodType = (kind: string) => {
+  switch (kind) {
+    case "date":
+      return "DATE_RANGE";
+    case "number":
+      return "NUMBER_RANGE";
+    case "string":
```

Comment:
We have also kind "boolean" which have values of type `string | boolean`. Tricky situation. You need to check on wiki cube with `isMinor` and our cubes with "booleans" as string.